src/view/eventItem.js

```diff
@@ -0,0 +1,101 @@
+import dayjs from 'dayjs';
+import { createElement } from '../render';
+
+const isFavorite = (data) => {
+  if(data) {
+    return 'active';
+  }
+};
+
+
+const generateEventDuration = (data) => {
+  const timeFrame = dayjs(data.dateTo).diff(dayjs(data.dateFrom), 'minutes');
+  return timeFrame;
```

Comment:
Привыкай использовать более короткую запись без промежуточного результата, если ты ничего с ним больше не делаешь
```suggestion
  return dayjs(data.dateTo).diff(dayjs(data.dateFrom), 'minutes');
```